lib/appium.js

```diff
@@ -344,32 +346,136 @@ class AppiumDriver extends BaseDriver {
     }
   }
 
+  pluginsToWrapCmd (cmd) {
+    return this.plugins.filter((plugin) => plugin.wrapCommands.includes(cmd));
```

Comment:
we need to check if `wrapCommands` property exists first. an invalid plugin should not crash the whole server

Reply:
actually i just need to delete this, it's obsolete